### PR TITLE
NISP-2025: Changed NI credits message layout

### PR DESCRIPTION
**Pull request by amyhart6**
With @fred-jackson

### CODE REVIEW

**Comment by amyhart6** [comment]
@dspork and @prashantchoudhari - only minor changes to the scala pages, and separation of the message into two parts

@swaistle and @InduTest - not sure that this will affect your tests and the messages haven't changed, just the layout.

File: conf/messages [hunk 1]
```diff
@@ -280,8 +280,10 @@ nisp.nirecord.gap.selfemployed.singular = Self-employment: <strong>{0} week</str
 nisp.nirecord.gap.selfemployed.plural = Self-employment: <strong>{0} weeks</strong>
 nisp.nirecord.gap.voluntary.singular = Voluntary: <strong>{0} week</strong>
 nisp.nirecord.gap.voluntary.plural = Voluntary: <strong>{0} weeks</strong>
-nisp.nirecord.gap.whenyouareclaiming.singular = National Insurance credits: <strong>{0} week</strong> (this week may have been added to your record if you were ill/disabled, unemployed, caring for someone full-time or on jury service) 
-nisp.nirecord.gap.whenyouareclaiming.plural= National Insurance credits: <strong>{0} weeks</strong> (these may have been added to your record if you were ill/disabled, unemployed, caring for someone full-time or on jury service) 
+nisp.nirecord.gap.whenyouareclaiming.singular = National Insurance credits: <strong>{0} week</strong>
```
Comment (roundrobin99) [inline conf/messages]: Should we remove the {0} and replace it with "1" (for all singular messages)? 

It is only for a single year, so we don't need to pass a value.

**Comment by fred-jackson** [comment]
@roundrobin99 - doh! That makes complete sense...

**Comment by InduTest** [comment]
Ran regression and updated acceptance messaging. Good to merge

**Pull request merged**